### ISSUE TITLE
Unable to edit & reload zipped mod

### ISSUE THREAD

**Issue by CuriousMike56** [RigsOfRods/rigs-of-rods#2452]
- Launch RoR and spawn a vehicle (keep it open)
- Open the vehicle's zip using **Windows File Explorer** (other zip managers such as 7-zip do not allow zip updates while RoR is opened), drag the `.truck` file onto your desktop
- Make any change
- Drag the `.truck` back into the zip, overwriting when prompted 
- Try to reload the vehicle:
![image](https://user-images.githubusercontent.com/46073351/66063341-26393980-e511-11e9-9126-e21356fda95d.png)

**Comment by only-a-ptr** [comment]
This is never going to work, when the ZIP bundle is in use by OGRE resource system, it's in use, and the fact MSWindows lets you make underhanded edits to it doesn't make any difference.

I reproduced it and this is what I got in RoR.log:
```
14:28:47: Ogre::FileNotFoundException::FileNotFoundException: Corrupted archive 'C:\Users\myusername\Builds\rigs-of-rods\bin\content\dafsemi.zip' in Ogre::`anonymous-namespace'::ZipArchive::open at C:\Users\myusername\.conan\data\OGRE\1.11.6-with-patches\anotherfoxguy\stable\build\5e54358c0c0ecfa815ddb9f685f86696c1ff2c91\OgreMain\src\OgreZip.cpp (line 265)
```
I don't have a Linux box for testing but I'm sure it simply wouldn't let me change the ZIP while loaded.

I understand you'd like to tune mods this way - I thought about it already, my idea is to introduce a new directory `MyGames\Rigsofrods\projects` which would only contain subdirectories and would be ignored by modcache - mods would be loaded by separate "projects" menu and always fully reloaded from disk, enabling modder to add/edit/remove resources on the go. I even wrote some code but it got obsolete: https://github.com/only-a-ptr/rigs-of-rods/tree/project-folders
